capstone-frontend/src/styles/Clubs.css

```diff
@@ -0,0 +1,47 @@
+.page-container {
+  text-align: center;
+  justify-content: center;
+  padding: 15px;
+  margin-top: 15px;
+}
+
+.title {
+  font-size: 50px;
+  text-align: center;
+  margin: 15px 0px;
+}
+
+.clubslist-container {
+  margin: 40px 0px;
+  justify-content: center;
+}
+
+#createclub {
+  color: #20bf6b;
+  font-weight: 500;
+  font-size: 16px;
+  background: #ffffff;
+  padding: 10px;
+  border: 4px solid #20bf6b;
+  transition: all 0.3s ease 0s;
+}
+
+#createclub:hover {
+  background: #20bf6b;
+  color: white;
+  border-radius: 50px;
+  border-color: #20bf6b;
+  transition: all 0.3s ease 0s;  
+  text-decoration: none;
+}
+
+.club-container {
+  margin: 10px;
+  padding: 5px;
+  border: 2px solid black;
+  text-align: center;
+}
+
+form div {
+  margin-bottom: 10px;
+}
```

Comment:
Can these names be abstracted to fit into styles for `Communities` too? Like `group-container` and `create-group` instead of `club-container` and `createclub`?

Additionally, I think it would be better if we stuck to `-` between each word to make it consistent. So like `create-club` instead of `createclub`
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Sure I can rename the css to be more general so when we make the Communities feature it can use the same styles.